datafusion/core/src/catalog/information_schema.rs

```diff
@@ -264,10 +244,7 @@ impl SchemaProvider for InformationSchemaProvider {
     }
 
     fn table_exist(&self, name: &str) -> bool {
-        matches!(
-            name.to_ascii_lowercase().as_str(),
-            TABLES | VIEWS | COLUMNS | SCHEMATA
-        )
+        INFORMATION_SCHEMA_TABLES.contains(&name)
```

Comment:
we probably want to preserve case insensitive behavior like it was before

Reply:
oh, thanks.